### PR TITLE
Bounds

### PR DESCRIPTION
**Pull request by PeterConstable**
Moves and edits content.

### CODE REVIEW

File: colr-gradients-spec.md [hunk 1]
```diff
@@ -1221,7 +1162,50 @@ table would not require the corresponding graph of paint tables to be
...
+
+NOTE: The bounding box of the base glyph can be used to allocate a drawing
+surface without needing to traverse the graph of the color glyph definition.
+
+A valid color glyph definition shall define a bounded region—that is, it shall
+paint within a region for which a finite bounding box could be defined. The
+different paint formats have different boundedness characteristics:
+
+* PaintGlyph is always inherently bounded.
+* PaintSolid, PaintLinearGradient and PaintRadialGradient are always unbounded.
```
Comment (rsheeter) [inline colr-gradients-spec.md:1195]: Optional: "are inherently unbounded", for consistency with PaintGlyph

File: colr-gradients-spec.md [hunk 2]
```diff
@@ -1221,7 +1162,50 @@ table would not require the corresponding graph of paint tables to be
...
+* PaintColrLayers is bounded *if and only if* all referenced sub-graphs are
+bounded.
+* PaintColrGlyph is bounded *if and only if* the color glyph definition for the
+referenced base glyph ID is bounded.
+* PaintTransformed, PaintTranslate, PaintRotate and PaintSkew are bounded *if
+and only if* the referenced sub-graph is bounded.
+* PaintComposite is either bounded or unbounded, according to the compositing or
+blending mode used. See 5.7.11.2.5.11 for details.
```
Comment (rsheeter) [inline colr-gradients-spec.md:1203]: nit: the referenced sub-graphs also factor in, that is it's boundedness is based on the sub-graphs *and* the mode.

Suggest deleting "compositing or blending" and and referencing compositeMode or composite mode to match the field and enumeration naming.

**Review by rsheeter — APPROVED** [review]

File: colr-gradients-spec.md (hunk 2 above)
Comment (PeterConstable) [reply colr-gradients-spec.md:1203]: Changing...

File: colr-gradients-spec.md (hunk 1 above)
Comment (PeterConstable) [reply colr-gradients-spec.md:1195]: Changing...

**Pull request merged**